src/backend/sev/data.rs

```diff
@@ -170,7 +170,7 @@ pub fn dev_sev() -> Datum {
 
 pub fn sev_enabled_in_kernel() -> Datum {
     let mut datum = Datum {
-        name: " SEV-SNP is enabled in host kernel with the v10 host patches".into(),
+        name: " SEV-SNP is enabled in host kernel".into(),
```

Comment:
We don't want to have notice about the SNP patches? Is v15 the last?

Reply:
We don't test for the specific patch version, so if the user has a different version installed, it might be misleading to say that they have that specific version installed.